developer-docs-site/docs/guides/system-integrators.md

```diff
@@ -0,0 +1,62 @@
+slug: "system-integrators"
+---
+
+# System Integrators Guide
+
+This guide will walk you through all you need to successfully integrate with the Aptos blockchain. This guide assumes familiarity with blockchains, command-line interfaces, and other concepts discussed throughout the developer site. Where possible links will be made to relevant topics.
+
+An integrator may follow some or all of these steps:
+* Create an account on the blockchain
+* Exchange account identifiers with another entity on the blockchain to perform swaps, for example
+* Create a transaction
+* Obtain a gas estimate and validate the transaction for correctness
+* Ask the user to verify the intent and cost of the transaction
+* Submit the transaction to the blockchain
+* Wait for the outcome of the transaction
+* Show the completed transaction to the user
+* The ability to see all interactions for a given account with a specific account -- that is withdraws and deposits
+
+## Accounts on Aptos
+
+An [account](../concepts/basics-accounts) represents a resource on the Aptos Blockchain that can send transactions. Each account is identified by a particular 32-byte account address and is a container for Move modules and Move resources. On Aptos, accounts must be created on-chain prior to any operations involving that account. With the caveat that the framework supports implicitly creating accounts when tranfering Aptos coin via `account::transfer`.
+
+Account addresses are defined at creation time as a one way function from the public key(s) and signature algorithm used for authentication for the account. This is covered in depth in [account](..concepts/basics-accounts) as well as demonstrated in the [Typescript SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/typescript/sdk/src/aptos_account.ts#L66) and [Python SDK](https://github.com/aptos-labs/aptos-core/blob/9b85d41ed8ef4a61a9cd64f9de511654fcc02024/ecosystem/python/sdk/aptos_sdk/account_address.py#L43) -- note currently both these SDKs only demonstrate how to generate an address from an Ed25519 single signer.
+
+An Account on Aptos has the ability to rotate keys so that potentially compromised keys cannot be used to access accounts. Refreshing keys has been generally regarded as good hygiene in the security field. This presents a challenge for system integrators who are familiar with using a mnemonic to represent both a private key and its associated account. Before launch, Aptos will have an on-chain mapping that makes to maintain this simplicity. The on-chain data maps an effective account address as defined by the current mnemonic to the actual account address. The details are forthcoming.
+
+Currently, Aptos only supports a single, unified identifier for an account. Accounts on Aptos are universally regarded as a 32-byte hex string. A hex string shorter than 32-bytes should be consider padded with leading zeroes, e.g., 0x1 => 0x0000000000000...01.
```

Comment:
`should be consider padded`, maybe my English level is not advanced enough to understand this :)

Reply:
hahaha, thanks, will revise :)